src/test/java/core/basesyntax/activity/ActivityStrategyImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.activity;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.handler.FruitHandler;
+import core.basesyntax.service.handler.RemoveFruitHandlerImpl;
+import core.basesyntax.service.handler.UpdateFruitHandlerImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ActivityStrategyImplTest {
+    private static final FruitHandler ADD_TO_STORAGE = new UpdateFruitHandlerImpl();
+    private static final FruitHandler TAKE_FROM_STORAGE = new RemoveFruitHandlerImpl();
+    private static final Map<Activities, FruitHandler> handlerMap = new HashMap<>();
+    private final ActivityStrategy activityStrategy = new ActivityStrategyImpl(handlerMap);
+
+    @BeforeClass
+    public static void fillMap() {
```

Comment:
```suggestion
    public static void setup() {
```
if you want, you can have private methods for such purposes